lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1619,6 +1610,12 @@ public function get_item_schema() {
 				'type'        => 'boolean',
 				'context'     => array( 'view', 'edit' ),
 			);
+
+			$schema['properties']['password'] = array(
+				'description' => __( 'A password to protect access to the content and excerpt.' ),
+				'type'        => 'string',
+				'context'     => array( 'edit' ),
+			);
```

Comment:
I'm curious whether anyone else agrees with me, then :) If not, I won't push.

Would we ever have content protected, but excerpt not?